### PR TITLE
[improve][doc] add flag instructions to pulsar-daemon

### PR DESCRIPTION
**Pull request by Anonymitaet**
This PR:
- Fixes https://github.com/apache/pulsar/issues/19512 (thanks @horizonzy for providing technical inputs!)
- Adds missing `/` to broker metrics doc (thanks @hangc0276 for raising this up!)

- [X] `doc` 

cc @momo-jun @DaveDuggins @D-2-Ed

### CODE REVIEW

File: static/reference/next/pulsar-daemon/pulsar-daemon.md
Comment (Anonymitaet) [inline static/reference/next/pulsar-daemon/pulsar-daemon.md:14]: Currently, this content can't be previewed through the normal preview steps (It might be caused by some technical issues. I'll record that in a GitHub issue). But it does not block merge this PR since the writing syntax is correct.

**Comment by Anonymitaet** [comment]
@hangc0276 @horizonzy could you please help review? Thank you!

File: docs/reference-metrics.md
Comment (horizonzy) [inline docs/reference-metrics.md:116]: Is there some difference between them? In HTTP protocol, the `/metrics` is equal to `/metrics/`

Comment (Anonymitaet) [reply docs/reference-metrics.md:116]: Hi @hangc0276 can you explain this?

Comment (Anonymitaet) [reply docs/reference-metrics.md:116]: Hi @horizonzy as we discussed (+@hangc0276) just now, users can not get data if using `/metrics` with `curl`, they need to use `/metrics/`.

(A workaround is to add `-L` to redirect, but we do not add this workaround to avoid making docs complicated)

Can you approve this PR if no more comments? Thank you!

Comment (horizonzy) [reply docs/reference-metrics.md:116]: Get it

**Review by horizonzy — APPROVED** [review]
LGTM.

**Review by hangc0276 — APPROVED** [review]

**Pull request merged**
